doc/guides/contributing/pull-requests.md

```diff
@@ -2,35 +2,35 @@
 
 * [Dependencies](#dependencies)
 * [Setting up your local environment](#setting-up-your-local-environment)
-  * [Step 1: Fork](#step-1-fork)
```

Comment:
I think the first word after a colon should be capitalized in a header, even if that's not how it works in a sentence. Here's [the APA style guide on sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case) (emphasis added):

> In sentence case, lowercase most words in a title or heading. Capitalize only the following words:
> 
> * the first word of the title or heading
> * the first word of a subtitle
> * ***the first word after a colon, em dash, or end punctuation in a heading***
> * nouns followed by numerals or letters
> * proper nouns (such as the names of racial or ethnic groups)

Reply:
understood. i have made the changes.